Moderation-Policy.md

```diff
@@ -191,10 +191,10 @@ of the [Code of Conduct][].
 * Moderation of Posts authored by non-Collaborators may result in those
   non-Collaborators being Blocked temporarily or indefinitely from further
   participation in the Node.js GitHub organization.
-* In the case where a GitHub Account appears to have been created with no
-  intention to collaborate in good faith, swift actions may be taken without
-  following the above procedures including: removing Posts, Blocking
-  indefinitely, and reporting accounts to GitHub.
+* In the case where a GitHub Account, Issue, or Pull Request appears to have been
+  created with no intention to collaborate in good faith, swift actions may be taken
+  without following the above procedures including: removing Posts, deleting Issues and
+  Pull Requests, Blocking indefinitely, and reporting accounts to GitHub.
```

Comment:
Non-blocking, but I'd prefer shorter, clearer constructs:

```markdown
You may bypass the above procedures if it is clear that there is no intention to
collaborate in good faith. In such cases, you may immediately:

* remove posts
* delete issues/pull requests
* block users
* report accounts to GitHub.
```

Reply:
I don't think we need to avoid personal pronouns like _you_ in the Moderation Policy the way we do in our technical documentation, but in case my use of _you_ rankles, how about this?:

```markdown
Bypass the above procedures if it is clear that there is no intention to
collaborate in good faith. In such cases, it is permissible to immediately:

* remove posts
* delete issues/pull requests
* block users
* report accounts to GitHub
```

Reply:
`one may immediately`

Reply:
(@MylesBorins If you want, I can edit the text myself if you don't object to my/@ljharb's suggestions.)